package.json

```diff
@@ -24,6 +23,8 @@
   },
   "homepage": "https://github.com/FACG6/AMiDEAST#readme",
   "devDependencies": {
-    "concurrently": "^4.1.0"
+    "concurrently": "^4.1.0",
+    "tap-spec": "^5.0.0",
```

Comment:
move these modules into ``` server/package.json ```

Reply:
No, I make it on purpose,
where is the problem in that

Reply:
because you will not use it on the client side, just on the server side , not in both